Example shell commands/Open the current file in vscode and jump to the current position.md

```diff
@@ -0,0 +1,4 @@
+# Example
+
+## Windows/Mac/Linux
+ - If you have the [Visual Studio Code] editor installed on your system (and the shell command available), you can easily jump between obsidian and VSCode using the following SC command: `code --goto {{file_position:absolute}}:{{caret_position}}`
```

Comment:
- I think VSCode link is missing
- file_position -> file_path
- obsidian -> capital O

```suggestion
 - If you have the [Visual Studio Code](https://code.visualstudio.com/) editor installed on your system (and the shell command available), you can easily jump between Obsidian and VSCode using the following SC command: `code --goto {{file_path:absolute}}:{{caret_position}}`
```

Reply:
Ahh that was sloppy, I wrote it up very quickly (too quickly it seems haha). Thanks for catching that!